data/campaigns/emp04.wmf/scripting/tribes/trainingcamp1/init.lua

```diff
@@ -89,88 +85,88 @@ tribes:new_trainingsite_type {
       sleep = {
          descname = "sleeping",
          actions = {
-            "sleep=5000",
-            "checksoldier=soldier attack 9", -- dummy check to get sleep rated as skipped - else it will change statistics
+            "sleep=duration:5s",
+            "checksoldier=soldier:attack level:9", -- dummy check to get sleep rated as skipped - else it will change statistics
```

Comment:
should be changed to return=skipped as this is the current state of the normal Trainingsites. This tribe specific building missed a couple of changes in the normal Trainingsites.
Howewver the programs should not incorporate the fixes to fail early as this building is intended to work poorly